mass_mailing_custom_unsubscribe/models/mail_mass_mailing_list.py

```diff
@@ -1,14 +0,0 @@
-# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-
-from odoo import fields, models
-
-
-class MailMassMailing(models.Model):
-    _inherit = "mail.mass_mailing.list"
-
-    not_cross_unsubscriptable = fields.Boolean(
```

Comment:
Why this feature has been removed? It's still needed in our customer.

Reply:
Check the PR description, it's explained there. :wink: